aws/config.ts

```diff
@@ -1,29 +1,69 @@
+import * as eks from "@pulumi/eks";
+import * as aws from "@pulumi/aws";
+import * as awsx from "@pulumi/awsx";
 import * as pulumi from "@pulumi/pulumi";
+import { ChartOpts } from "@pulumi/kubernetes/helm/v3";
 
 const config = new pulumi.Config();
 
-// couldn't get generics working here with `config.get<T>(key)`
-function getStringOrDefault(key: string, def: string): string {
-  let ret = config.get<string>(key);
-  if (ret == undefined) {
-    ret = def;
+function getBooleanOrDefault(key: string, def: boolean): boolean {
+  const ret = config.getBoolean(key);
+  return ret == undefined ? def : ret;
+}
+
+export const LightbendNamespace = "lightbend";
+
+export const vpcArgs: awsx.ec2.VpcArgs = {
```

Comment:
I'm trying to define most of the configurations in terms of what Pulumi already provides. This looks better than having primitive types.